taccsite_cms/settings.py

```diff
@@ -289,6 +289,8 @@ def gettext(s): return s
     ('ui', os.path.join(BASE_DIR, 'taccsite_ui', 'dist')),
 )
 
+STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
```

Comment:
This seems to [become deprecated in Django 4.2](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-STORAGES), which we want to use soon. If so, would you recommend we note that somewhere ([Jira](https://jira.tacc.utexas.edu/browse/TUP-454)) (code comment here)? _The new way (not available until 4.2) would be [via `STORAGES` setting](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-STORAGES)._